EventKit/eventkit-cloud
# fixes 3857 feature server sources

# Description of Improvement

Allows the use of arcgis feature services that are in 3857 projection.

## How to test

Use an arcgis feature service which is in the 3857 projection and ensure that the data gets pulled. 

## Quality Assurance

The following items have been updated:


## Screenshots



:smile:
